app/src/index.ts

```diff
@@ -158,7 +158,12 @@ function restart() {
 app.on('before-quit', () => {
   if (proc) {
     proc.off('exit', restart)
-    proc.kill()
+    proc.kill('SIGINT') // send SIGINT signal to the server, which also stop and loaded llms
+    setTimeout(() => {
```

Comment:
This stops a llama.cpp server that is running when the mac app exits.

Reply:
The `setTimeout` may never get called since the app exists and the v8 runtime shuts down